src/databricks/labs/ucx/install.py

```diff
@@ -170,6 +173,16 @@ def deploy_schema(sql_backend: SqlBackend, inventory_schema: str):
     deployer.deploy_view("table_estimates", "queries/views/table_estimates.sql")
 
 
+def load_cluster_specs() -> dict[str, ClusterSpec]:
```

Comment:
Don't load something from a yaml file if it can be defined as 5 lines of python code. Revert